File: Leetcode/2032/2032.cpp
class Solution {
public:
    vector<int> twoOutOfThree(vector<int>& nums1, vector<int>& nums2, vector<int>& nums3) {
        constexpr int MAXN = 105;
        bitset<MAXN> bs1, bs2, bs3;
        for (auto &i : nums1) bs1[i] = true;
        for (auto &i : nums2) bs2[i] = true;
        for (auto &i : nums3) bs3[i] = true;
        vector<int> vec;
        int counter;
        for (int i = 0; i < MAXN; ++i) {
            counter = 0;
            if (bs1[i]) ++counter;
            if (bs2[i]) ++counter;
            if (bs3[i]) ++counter;
            if (counter >= 2) vec.emplace_back(i);
        }
        return vec;
    }
};

File: Leetcode/2032/2032.go
func twoOutOfThree(nums1 []int, nums2 []int, nums3 []int) []int {
	const N = 105
	vis1 := [N]bool{}
	vis2 := [N]bool{}
	vis3 := [N]bool{}
	for i := 0; i < len(nums1); i++ {
		vis1[nums1[i]] = true
	}
	for i := 0; i < len(nums2); i++ {
		vis2[nums2[i]] = true
	}
	for i := 0; i < len(nums3); i++ {
		vis3[nums3[i]] = true
	}
	var ans []int
	for i := 0; i < N; i++ {
		if (vis1[i] && vis2[i]) || (vis1[i] && vis3[i]) || (vis3[i] && vis2[i]) {
			ans = append(ans, i)
		}
	}
	return ans
}